uint8 UNKNOWN=0
uint8 ROAD=1
uint8 TRAFFIC_SIGN=2
uint8 TRAFFIC_LIGHT=3
uint32 LaneletID	#在哪个车道
uint32 CenterpointID	#在车道的具体位置
uint8  SignType		#类型


